package.json

```
@@ -26,7 +26,8 @@
     "node-fetch": "^2.6.0"
   },
   "scripts": {
-    "build": "react-scripts build && cp CNAME build",
+    "build": "react-scripts build && cp CNAME build && cp public/googleb1123845f3bf5d79.html build
```

Yup - that should add it, although this would help shorten it up a bit:
```suggestion
    "build": "react-scripts build && cp CNAME public/googleb1123845f3bf5d79.html build
```